model/src/main/java/org/cbioportal/genome_nexus/model/MutationAssessor.java

```diff
@@ -1,261 +1,76 @@
-        this.mappingIssue = mappingIssue;
-    }
-
-    @Field("var")
-    public String getVariant() {
-        return variant;
-    }
-
-    public void setVariant(String variant) {
-        this.variant = variant;
-    }
-
-    @Field("uprot")
+    
     public String getUniprotId() {
         return uniprotId;
     }
-
+    
     public void setUniprotId(String uniprotId) {
         this.uniprotId = uniprotId;
     }
-
-    @Field("rsprot")
-    public String getRefseqId() {
-        return refseqId;
-    }
-
-    public void setRefseqId(String refseqId) {
-        this.refseqId = refseqId;
-    }
-
-    @Field("gaps")
-    public Double getMsaGaps() {
-        return msaGaps;
-    }
-
-    public void setMsaGaps(Double msaGaps) {
-        this.msaGaps = msaGaps;
-    }
-
-    @Field("msa_height")
-    public Integer getMsaHeight() {
-        return msaHeight;
-    }
-
-    public void setMsaHeight(Integer msaHeight) {
-        this.msaHeight = msaHeight;
-    }
-
-    @Field("chr")
-    public String getCodonStartPosition() {
-        return codonStartPosition;
-    }
-
-    public void setCodonStartPosition(String codonStartPosition) {
-        this.codonStartPosition = codonStartPosition;
-    }
-
-    @Field("up_pos")
-    public Integer getUniprotPosition() {
-        return uniprotPosition;
-    }
-
-    public void setUniprotPosition(Integer uniprotPosition) {
-        this.uniprotPosition = uniprotPosition;
-    }
-
-    @Field("up_res")
-    public String getUniprotResidue() {
-        return uniprotResidue;
-    }
-
-    public void setUniprotResidue(String uniprotResidue) {
-        this.uniprotResidue = uniprotResidue;
-    }
-
-    @Field("rs_pos")
-    public Integer getRefseqPosition() {
-        return refseqPosition;
-    }
-
-    public void setRefseqPosition(Integer refseqPosition) {
-        this.refseqPosition = refseqPosition;
-    }
-
-    @Field("rs_res")
-    public String getRefseqResidue() {
-        return refseqResidue;
-    }
-
-    public void setRefseqResidue(String refseqResidue) {
-        this.refseqResidue = refseqResidue;
-    }
-
-    @Field("cnt_cosmic")
-    public Integer getCosmicCount() {
-        return cosmicCount;
-    }
-
-    public void setCosmicCount(Integer cosmicCount) {
-        this.cosmicCount = cosmicCount;
-    }
-
-    @Field("cnt_snps")
-    public Integer getSnpCount() {
-        return snpCount;
-    }
-
-    public void setSnpCount(Integer snpCount) {
-        this.snpCount = snpCount;
+    
+    private Integer sv;
+    
+    public Integer getSv() {
+        return sv;
+    }
+    
+    public void setSv(Integer sv) {
+        this.sv = sv;
+    }
+    
+    private String hgvspShort;
+    
+    public String getHgvspShort() {
+        return hgvspShort;
+    }
+    
+    public void setHgvspShort(String hgvspShort) {
+        this.hgvspShort = hgvspShort;
+    }
+    
+    private Double f_score;
+    
+    public Double getF_score() {
+        return f_score;
+    }
```

Comment:
Not sure if it is a big deal but previously we did something like this to have nicer Java field names.

```Java
@Field("F_score")
public Double getFunctionalImpactScore() {
    return functionalImpactScore;
}
```

Should we do something similar for the new model?

Reply:
@onursumer Good point! Right now I just use whatever in their original data, I guess we can rename to `functionalImpact` and `functionalImpactScore`